torchvision/models/vgg.py

```diff
@@ -60,13 +66,14 @@ def _initialize_weights(self):
                 nn.init.constant_(m.bias, 0)
 
 
-def make_layers(cfg, batch_norm=False):
+def make_layers(cfg: List[Union[str, int]], batch_norm: bool = False) -> nn.Sequential:
     layers = []
```

Comment:
mypy seems to be complaining here. You might need to annotate this with `List[nn.Module]` maybe?

Reply:
But doing so would trigger another error on `VGG`constructor that expects a `features: nn.Module`, wouldn't it?

Reply:
I think @fmassa meant annotating `layers: List[nn.Module]`. Since this is a local variable, the constructor has nothing to do with it.

Reply:
Oh my bad, I thought he was referring the function definition line. This seems to do the trick, just pushed the edit!